examples/httpbin-demo/tests/integration/test_charm.py

```diff
@@ -12,34 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import asyncio
 import logging
 from pathlib import Path
 
-import pytest
+import jubilant
 import yaml
-from pytest_operator.plugin import OpsTest
 
 logger = logging.getLogger(__name__)
 
 METADATA = yaml.safe_load(Path('./charmcraft.yaml').read_text())
 APP_NAME = METADATA['name']
 
 
-@pytest.mark.abort_on_fail
-async def test_build_and_deploy(ops_test: OpsTest):
+def test_build_and_deploy(charm: Path, juju: jubilant.Juju):
     """Build the charm-under-test and deploy it together with related charms.
 
     Assert on the unit status before any relations/configurations take place.
     """
     # Build and deploy charm from local source folder
-    charm = await ops_test.build_charm('.')
     resources = {'httpbin-image': METADATA['resources']['httpbin-image']['upstream-source']}
 
     # Deploy the charm and wait for active/idle status
-    await asyncio.gather(
-        ops_test.model.deploy(charm, resources=resources, application_name=APP_NAME),
-        ops_test.model.wait_for_idle(
-            apps=[APP_NAME], status='active', raise_on_blocked=True, timeout=1000
-        ),
-    )
+    juju.deploy(f'./{charm}', app=APP_NAME, resources=resources)
+    juju.wait(jubilant.all_active)
```

Comment:
This isn't a perfect translation, as it's missing the `raise_on_blocked=True` bit. Does that actually matter? Does this charm Go into blocked? If so, we could do:

```python
juju.wait(jubilant.all_active, error=jubilant.any_blocked)
```